Libraries/LibGUI/GMenu.cpp

```diff
@@ -113,6 +113,11 @@ int GMenu::realize_menu()
             request.menu.enabled = true;
             request.menu.checkable = false;
             request.menu.checked = false;
+            
+            // no shortcut on submenu, make sure this is cleared out
```

Comment:
This looks like a useful fix, but please make it in a separate commit.

Reply:
Got this broken out now.